Frontend/lib/data/models/models.dart

```diff
@@ -1,9 +1,12 @@
 export 'address_info.dart';
 export 'address_list.dart';
+export 'broadcast_transaction.dart';
+export 'create_transaction.dart';
 export 'generate_address.dart';
 export 'get_all_transaction.dart';
 export 'get_new_address.dart';
 export 'total_utxo.dart';
+export 'total_utxo.dart';
```

Comment:
Why did you export total_utxo again, you already exported it

Reply:
That must be a typo, I will make the correction.